Add concept pages for Scanner and Healing (#1192)
minio/docs: source/glossary.rst

     MinIO combats bit rot with :term:`hashing` and :term:`erasure coding`.

   bit rot healing

>      Objects corrupted due to bit rot automatically restore to a healed state at time of read during any ``GET`` operation.

Reviewer: maybe?
```suggestion
     Objects corrupted due to bit rot are automatically healed the next time they are read by a ``GET`` operation.
```
could link to healing term as well
